### ISSUE TITLE
Failed to parse output of `cargo metadata`

### ISSUE THREAD

**Issue by wcampbell0x2a** [dtolnay/cargo-docs-rs#5]
```
$ git clone https://github.com/wcampbell0x2a/backhand.git
$ cd backhand
$ cargo --version
cargo 1.76.0-nightly (1aa9df1a5 2023-12-12)
$ cargo docs-rs --open
Error: Failed to parse output of `cargo metadata`

Caused by:
    invalid type: string "all", expected a sequence at line 1 column 632149
```

**Comment by dtolnay** [comment]
Fixed in 0.1.8.

**Comment by wcampbell0x2a** [comment]
> Fixed in 0.1.8.

Thanks!
